lib/std/crypto/25519/scalar25519.zig

```diff
@@ -0,0 +1,185 @@
+const std = @import("std");
```

Comment:
Same as above, would it be possible to name this module `scalar.zig`?

Reply:
Renamed!